### PR TITLE
Ride status

### PR DESCRIPTION
**Pull request by mly32**
### Summary 
image

Made ride status and progress bar.

### Test Plan 

Go to a particular day (I use Feb 22, 2021) and click on an event.

Change the ride status and late status of the progress bar to view how different progress bars look like.

### Breaking Changes  

<!-- - Database schema change (anything that changes DynamoDB document structure)
<!-- - Other change that could cause problems (Detailed in notes)

### CODE REVIEW

File: frontend/src/components/RideStatus/SModal.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            <div className={styles.bar}>
+              <ProgressBar status={ride.status} late={ride.late} />
+            </div>
+            <div className={styles.row}>
+              <div className={styles.column}>
+                <p>{moment(new Date(ride.startTime)).format('h:mm a')}</p>
+                <div className={styles.location}>
+                  <span className={cn(styles.tag, styles[ride.startLocation.tag || ''])}>{ride.startLocation.tag}</span>
+                  <p>{ride.startLocation.name}</p>
+                </div></div>
```
Comment (mg2239) [inline frontend/src/components/RideStatus/SModal.tsx:55]: can you make this more responsive for longer addresses? also add the gray bg like in the design
image

Comment (mg2239) [reply frontend/src/components/RideStatus/SModal.tsx:55]: this is 2/27 btw

**Review by mg2239 — CHANGES_REQUESTED** [review]

File: frontend/src/components/RideStatus/SModal.tsx (hunk 1 above)
Comment (mly32) [reply frontend/src/components/RideStatus/SModal.tsx]: Nice catches. 👍

**Comment by mly32** [comment]
completed.

**Review by mg2239 — APPROVED** [review]
looks great!

**Pull request merged**